src/main/java/org/fcrepo/spec/testsuite/versioning/LdprmHttpOptions.java

```diff
@@ -52,16 +58,27 @@ public void ldprmMustSupportOptions(final String uri) {
                                         ps);
 
         //create an LDPRm
+        final String mementoURI = createMemento(uri, info);
+        final Response mementoResponse = doGet(mementoURI);
         //verify that a subsequent GET returns OPTIONS in "Allow" header.
+        confirmPresenceOfHeaderValueInMultiValueHeader("Allow", "OPTIONS", mementoResponse);
+    }
 
+    private String createMemento(final String uri, final TestInfo info) {
```

Comment:
I think something like this should go into the AbstractVersioningTest.java file.  But, I don't think it should do things - namely create a versioned resource and create a memento.  I'd say pass in the URI of the versioned resource, and only have this create the memento.  

I'd like to use this function in my PR, hence wanting it up in the Abstract Versioning class.  It's pretty handy.

Reply:
@bseeger 👍